pkg/local_object_storage/blobstor/fstree/fstree_write_linux.go

```diff
@@ -243,3 +243,42 @@ func (w *linuxWriter) writeFile(p string, data []byte) error {
 	}
 	return nil
 }
+
+func (w *linuxWriter) writeBatch(objs []putPrm) error {
+	w.batchLock.Lock()
+	defer w.batchLock.Unlock()
```

Comment:
No need to even look at this batch, just forget about it. Create a batch of your own and work with it here.